simul/cothority/cothority.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"flag"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/sda"
+	// Empty imports to have the init-functions called which should
+	// register the protocol
+	"github.com/dedis/cothority/lib/monitor"
+	_ "github.com/dedis/cothority/protocols"
+	"github.com/dedis/cothority/protocols/manage"
+	"time"
+)
+
+/*
+Cothority is a general node that can be used for all available protocols.
+*/
+
+// The address of this host - if there is only one host in the config
+// file, it will be derived from it automatically
+var HostAddress string
+
+// ip addr of the logger to connect to
+var Monitor string
+
+// Simul is != "" if this node needs to start a simulation of that protocol
+var Simul string
```

Comment:
privatised a lot of variables. As this structure is written/read using toml, the elements that have to be written/read need to be public.